README.md

```diff
@@ -346,7 +346,9 @@ The privacy properties of shared storage are enforced through limited output. So
 
 The worklet selects from a small list of (up to 8) URLs, each in its own dictionary with optional reporting metadata. The chosen URL is stored in an opaque URL that can only be read within a [fenced frame](https://github.com/shivanigithub/fenced-frame); the embedder does not learn this information. The chosen URL represents up to log2(num urls) bits of cross-site information. The URL must also be k-anonymous, in order to prevent much 1p data from also entering the Fenced Frame. Once the Fenced Frame receives a user gesture and navigates to its destination page, the information within the fenced frame leaks to the destination page. To limit the rate of leakage of this data, there is a bit budget applied to the output gate. If the budget is exceeded, the selectURL() will return the default (0th index) URL.
 
-selectURL() is disallowed in nested Fenced Frames. This is to prevent leaking lots of bits all at once via selectURL() chaining (i.e. a fenced frame can call selectURL() to add a few more bits to the fenced frame's current URL and render the result in a nested fenced frame). Though chaining seems quite useful, and we intend to revisit this. Use cases that will benefit from selectURL() being allowed from inside the top level fenced frame: [issue](https://github.com/WICG/fenced-frame/issues/44).
+selectURL() is disallowed in nested Fenced Frames. This is to prevent leaking lots of bits all at once via selectURL() chaining (i.e. a fenced frame can call selectURL() to add a few more bits to the fenced frame's current URL and render the result in a nested fenced frame). Use cases that will benefit from selectURL() being allowed from inside the top level fenced frame: [issue](https://github.com/WICG/fenced-frame/issues/44).
```

Comment:
I don't quite understand what is being proposed here. Are you allowing selectURL() within selectURL fenced frames?

Reply:
Yes. selectURL() within top level selectURL FFs are allowed. By "nested FF" I intended to mean a FF tree with depths >= 2.

Reply:
This is very subtle. Perhaps make it super clear what the limitation is? Something like:

selectURL() can be called in a top-level fenced frame, but not from within a nested fenced frame.

Reply:
Done.